xml/System.Windows.Forms/MdiClient.xml

```diff
@@ -86,7 +86,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the background image displayed in the <see cref="T:System.Windows.Forms.MdiClient" /> control.</summary>
-        <value>An <see cref="T:System.Drawing.Image" /> that represents the image to display in the background of the control.</value>
+        <value>An image that represents the image to display in the background of the control.</value>
```

Comment:
```suggestion
        <value>The image to display in the background of the control.</value>
```

Reply:
You beat me to it :) I didn't see your suggestion because GitHub does not refresh sometimes. I'll remove my suggestion.